src/stores/Create.ts

```diff
@@ -393,18 +547,31 @@ export const getInitialCreateState = (
                 produce((state) => {
                     const collections: string[] = [];
                     const configs = {};
+                    const { resourceSchema } = get();
 
                     value.forEach((collection) => {
                         collection.writes_to.forEach((writes_to) => {
                             collections.push(writes_to);
-                            configs[writes_to] = getDefaultJsonFormsData();
+                            configs[writes_to] =
+                                createJSONFormDefaults(resourceSchema);
```

Comment:
Need to set the actual defaults of a config so that if there are multiple bindings added on load then they have errors properly set.